server/grpc/filter.go

```diff
@@ -39,9 +42,11 @@ func createFilterFunc(agentFilter rpc.Filter) queue.FilterFn {
 				continue
 			}
 
-			if taskLabelValue != agentLabelValue {
-				return false
+			match, err := doublestar.Match(taskLabelValue, agentLabelValue)
```

Comment:
Could you give an example how / for what you would use a glob pattern here. Should also be added to the docs.

Reply:
I think this pull is already much I'll move that onto the next one

Reply:
-> #4122